Project 1/styles.css

```
@@ -0,0 +1,20 @@
+.card{
+        max-width: 800px;
+        margin: 30px auto;
+        background-color: #ffffff;}
+
+img{
+    max-width: 100%;}
+    title {
+        font-family: 'Slabo', serif;
+
+}
+body{    
+    font-family: 'Roboto', sans-serif;   }
+p{margin-left: 10; margin-left: 10px;}
+
+h1 h2 h3 h4 h5 h6
```

In order to target each of these selectors, they must be separated by commas like so:
```suggestion
h1, h2, h3, h4, h5, h6
```